.ast-grep/multicast-rule-check-test-documentation.yml

```
@@ -0,0 +1,156 @@
+        inside:
+          kind: expression_statement
+          inside:
+            kind: block
+            has:
+              kind: expression_statement
+              all:
+                - has:
+                    kind: identifier
+                - not:
+                    has:
+                        kind: function_definition
```

yamllint (1.35.1)

[error] wrong indentation: expected 22 but found 24

(indentation)